db/migrate/20160919080348_add_current_organization_id_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrentOrganizationIdToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :current_organization_id, :integer
```

Comment:
@yunixon I think we don't need this. Selection organization for which get data is frontend part. Now this store it in cookies. And API client will decide for which organization he want to get data.